### PR TITLE
build: fix pypi package include files

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-# Patterns to exclude from any directory
-global-exclude *~
-global-exclude *.pyc
-global-exclude *.pyo
-global-exclude .git
-global-exclude .github
-global-exclude .ipynb_checkpoints
-global-exclude __pycache__
-global-exclude tests
+include MANIFEST.in
```
Comment (aarnphm) [inline MANIFEST.in:3]: I don't think you have to include MANIFEST.in into sdist

File: MANIFEST.in [hunk 2]
```diff
@@ -30,3 +25,15 @@ prune typings
 prune scripts
 prune hooks
 prune docs
+prune .github
+prune bentoml/testing
```
Comment (aarnphm) [inline MANIFEST.in:29]: We actually offer some testing functions that users can use, @bojiang can you correct me on this?

File: .gitignore [hunk 3]
```diff
@@ -54,22 +54,7 @@ coverage.xml
...
-local_settings.py
-db.sqlite3
-
-# Flask stuff:
-instance/
-.webassets-cache
-
-# Scrapy stuff:
-.scrapy
+.xml
```
Comment (aarnphm) [inline .gitignore:57]:
```suggestion
*.xml
```

File: MANIFEST.in (hunk 2 above)
Comment (parano) [reply MANIFEST.in:29]: Those are meant to be used in e2e tests?

Comment (aarnphm) [reply MANIFEST.in:29]: yes, but then users can also use it as well iirc from my last conversation with bojiang

**Pull request merged**